packages/init/src/SliceMachineInitProcess.ts

```diff
@@ -965,6 +968,20 @@ ${chalk.cyan("?")} Your Prismic repository name`.replace("\n", ""),
 		]);
 	}
 
+	protected setDefaultMasterLocale(): Promise<void> {
+		return listrRun([
+			{
+				title: `Setting default master language...`,
```

Comment:
💡 I don't think we are talking about `master` language in the product, no?

See the title of the PB onboarding step: `What's your main content language?` 

I would do something like:

```suggestion
				title: `Setting main content language...`,
```

And also:

```
task.title = `Main content language set to ${chalk.cyan(
  "English - United States",
)} 🇺🇸. You can change it anytime in your project settings.`;
```

I would also use the label, as in the select in PB.

Reply:
Ok, will confirm the change of en-Us to the label with Come

Reply:
message change confirmed with Come, updated